sympy/matrices/eigen.py

```diff
@@ -288,6 +288,8 @@ def _eigenspace(M, eigenval, iszerofunc=_iszero, simplify=False):
 
 def _eigenvects_DOM(M, **kwargs):
     DOM = DomainMatrix.from_Matrix(M, field=True, extension=True)
+    DOM.to_dense()
```

Comment:
Surely this doesn't do anything because it would need to be `DOM = DOM.to_dense()`

Reply:
Yeah, I forgot to do that my bad. It passed all tests, so I ended up overlooking this mistake.